docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
+
+**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+
+In Filecoin, users pay storage miners to store their files. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
+
+![Diagram showing a four-step process for storing data with Filecoin. First step: User pays miner to store their data. Second step: Miner commits publicly via Filecoin’s blockchain to storing the data. Third step: The network constantly verifies that miners are storing data correctly. Fourth step: User pays miner to retrieve data.](./images/what-is-filecoin.png)
+
+## For users
+
+Filecoin provides users assurance their files have been stored safely and will be available in the future.
+
+Users can choose their preferred trade-off between cost, redundancy and speed by selecting the particular miners which store their files, or let software decide on their behalf. Applications that implement Filecoin can negotiate storage with any miner on the network. Unlike centralized storage systems, there is no need to implement a different API for each provider.
+
+At any time, users can verify that their files are being stored correctly by looking at proofs on Filecoin’s blockchain. This blockchain also powers Filecoin’s payment network and token, which users pay for storage with. Users don’t have to deal with the friction or fees associated with centralized payment networks.
```

Comment:
I don't agree that "centralized payment networks" would stress me out more with friction or fees. Entering my credit card number on Dropbox's website doesn't feel like friction to me compared to trading USD for a cryptocurrency and using it from the command line to make deals, but I may be missing the point you're making here. 

```suggestion
At any time, users can verify that their files are being stored correctly by looking at proofs on Filecoin’s blockchain. This blockchain also powers Filecoin’s payment network and token, which are used to pay for storage. Users don’t have to deal with the friction or fees associated with centralized payment networks.
```

Reply:
I’ve removed the sentence about friction and fees.